login.php

```diff
@@ -66,6 +68,7 @@
     </div>
 
     <input class="px-3 py-2 mt-4 text-lg bg-indigo-400 hover:bg-indigo-700 hover:text-white transition inline-block rounded-sm" type="submit" value="Login" />
+    <p><?php echo $errorMessage ?></p>
```

Comment:
the $errorMessage variable has `<p>` tags inside it, so you dont need another set around the echo